src/game/mod.rs

```diff
@@ -27,11 +29,12 @@ impl Game {
             score: 0,
             glyph_cache: GlyphCache::new("./assets/FiraSans-Regular.ttf").unwrap(),
             window_size: window_size,
+            asteroid_timer: 1.0,
+            asteroid_timer_max: 10.0,
```

Comment:
We can do that. Maybe starting at 6 seconds and then ramping up by .15 instead of .2